administrator/language/en-GB/plg_workflow_category.ini

```
@@ -0,0 +1,15 @@
+; Joomla! Project
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_WORKFLOW_CATEGORY_DESC="A plugin to assign categories during workflow transitions."
+PLG_WORKFLOW_CATEGORY_FIELD_CATEGORY_LABEL="Category"
+PLG_WORKFLOW_CATEGORY_FIELD_CATEGORY_DESC="Select the category to assign during this transition."
+PLG_WORKFLOW_CATEGORY_LABEL="Select Category"
+PLG_WORKFLOW_CATEGORY_NO_PRIMARY_KEY="No valid primary keys found for the selected category."
```

the word valid is redundant here as presumably it didnt find any invalid keys either
and as its only for a category (singlular) then the key should be singular as well

```suggestion
PLG_WORKFLOW_CATEGORY_NO_PRIMARY_KEY="No primary key found for the selected category."
```